components/pricing_modal/content.tsx

```diff
@@ -139,6 +155,26 @@ function Content(props: ContentProps) {
         formatMessage({id: 'admin.billing.subscription.planDetails.features.mfa', defaultMessage: 'Multi-Factor Authentication (MFA)'}),
     ];
 
+    // Default professional price
+    const defaultProfessionalPrice = 10;
+    const [professionalPrice, setProfessionalPrice] = useState(defaultProfessionalPrice);
+
+    const updateProfessionalPrice = (newIsMonthly: boolean) => {
+        if (!monthlyProfessionalProduct) {
+            return;
+        }
```

Comment:
Should this be toggling regardless if it's monthly or annually at any time?

Reply:
Initially, I had done everything based on having a monthly product and using that to find the yearly product. So, it wouldn't have been able to do anything if it didn't have the monthly product. 

But since I have to change some functions due to both products having the same sku, this might change as well.